tsconfig-lint.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Propably there is no need for a separate tsconfig for linting. Try using the main tsconfig and if it works remove this one.